---
permalink: /
title: ""
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

# About me

I am a PhD student in Computational Molecular Biology at the [Center for Computational Molecular Biology at Brown](https://ccmb.brown.edu) advised by [Lorin Crawford, Ph.D.](http://www.lcrawlab.com/) and [Jeffrey Bailey, Ph.D, MD](https://baileylab.org/). My research interests are broadly at the **intersection of computational modeling, computer science, statistics, and biology**. Specifically, I am interested in **developing and applying mathematical models and machine learning methods to analyze complex biological data to understand disease mechanisms and the underpinnings of drug resistance**.  I have experience with researching both cancer and malaria.

--- 
# Key research projects

## Bayesian Regression for Predicting the Spread and Selection of Mutations Causing Antimalarial Drug Resistance
Since 2014, resistance to Artemisinin combination therapies—the most common treatment for malaria—has been emerging in East Africa.  Artemisinin partial resistance (ART-R) is caused by mutations in the *Plasmodium falciparum* Kelch13 protein (K13) propeller domain. While the presence of K13 mutations driving this resistance is known, their selection dynamics remain unclear. To address this, I developed a Bayesian regression model that analyzes genomic data to estimate the spread of resistance-associated mutations and predict future trends. This model provides insights into how these mutations propagate across regions, enabling more precise forecasts and supporting public health strategies to mitigate the growing threat of drug-resistant malaria. By combining computational modeling and genomic data, this work contributes to safeguarding the efficacy of antimalarial treatments.

## A Statistical and Machine Learning Approach to Investigating Unmapped Regions of the Malaria Parasite Genome
While the malaria parasite's genome (*Plasmodium falciparum*) shows limited overall genetic variation, much of it is concentrated in highly dynamic subtelomeric regions. These regions encode genes that help the parasite evade human immunity, making them crucial for its survival. I am developing an AI-driven framework that leverages statistical methods and Variational Autoencoders to analyze these challenging genomic regions. This approach aims to uncover their role in shaping parasite population structure and identify shared genetic variants across populations. Insights from this research could improve our understanding of malaria progression and guide the development of novel drug targets.

## A Mathematical Approach to Understanding the Role of Human Mobility in the Spread of Antimalarial Drug Resistance
Resistance to Artemisinin Combination Therapies (ACTs) is spreading in malaria-endemic regions, driven by parasites with K13 mutations. I am developing a Bayesian model to estimate the movement of ACT-resistant infections between regions via humans and mosquitoes. By integrating mutation prevalence and selection dynamics, this work identifies high-risk regions and predicts how resistance spreads via mobility. These insights will guide public health strategies to combat drug-resistant malaria and improve global intervention efforts.

## Pancancer evolutionary model analyzing scRNAseq data
While at Dana-Farber, I developed an evolutionary modeling framework aimed at characterizing the progression of cancer across various tumor types. This project focused on analyzing copy number variations from single-cell RNA sequencing data to estimate distinct evolutionary modes in tumors. By applying machine learning techniques, the model identified patterns of clonal evolution and adaptive mechanisms used by cancer cells, providing valuable insights into tumor heterogeneity. The findings contributed to a better understanding of how tumors evolve over time and respond to different treatment pressures.

