---
permalink: /
title: ""
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

# About me

I am a PhD student in Computational Molecular Biology at the [Center for Computational Molecular Biology at Brown](https://ccmb.brown.edu) advised by [Lorin Crawford, Ph.D.](http://www.lcrawlab.com/) and [Jeffrey Bailey, Ph.D, MD](https://baileylab.org/). My research interests are broadly at the **intersection of computational modeling, computer science, statistics, and biology**. Specifically, I am interested in **developing machine and deep learning methods that address the key limitations of applying complex models to disease setting with small sample sizes and limited domain knowledge, with the potential of scaling the models to larger sample sizes**. Additionally, I hope to use the models to understand and evaluate **disease progression and drug resistance**.

--- 

# Key research projects

## Bayesian Regression for Predicting the Spread of Antimalarial Drug Resistance

I developed a Bayesian regression model to estimate the spread of antimalarial drug resistance mutations and predict future trends in resistance. This work utilized cross-sectional genomic data to infer the selection dynamics of specific resistance-associated mutations. The model enabled a more accurate understanding of how resistance mutations propagate across different regions, providing valuable forecasts that could inform public health responses and intervention strategies to prevent an arising public health threat.

## Machine Learning for Unmappable Gene Prediction in Malaria Parasites

I am building a machine learning model in Python to predict the significance of genes in malaria parasites that cannot be mapped to the reference genome. The machine learning approach helps evaluate how much these unmappable genes contribute to understanding parasite mobility and potential drug resistance mechanisms, thereby improving the overall understanding of malaria biology.

## Evolutionary Modeling of Cancer Progression at Dana-Farber Cancer Institute

While at Dana-Farber, I developed a evolutionary modeling framework aimed at characterizing the progression of cancer across various tumor types. This project focused on analyzing copy number variations from single-cell RNA sequencing data to estimate distinct evolutionary modes in tumors. By applying machine learning techniques, the model identified patterns of clonal evolution and adaptive mechanisms used by cancer cells, providing valuable insights into tumor heterogeneity. The findings contributed to a better understanding of how tumors evolve over time and respond to different treatment pressures.

