---
layout: archive
title: "People"
redirect_from: 
  - /people/
  - /people.html
---

# Project Lead/Coordinating Committee

List yourself

## Ben Tully

**Email:** tully.bj@gmail.com

**Twitter:** @phatombugs

**[Website](https://bjtully.github.io/)**

**[Google Scholar](https://scholar.google.com/citations?user=UIACF0wAAAAJ&hl=en)**

**Bio:** I am a microbiologist/bioinformatian working in the marine environment. My interests involve studying microbial communities and functions through the tool of metagenomics. As part of this research, I work to develop new bioinformatic tools that capitalizes on the growing number of environmental microbial genomes.

# Instructors

Please add yourself. See example(s) below.


## Jake L. Weissman

<img align="left" src="/images/jlw2.jpg">

**Email:** jakeweis@usc.edu

**Twitter:** @jlw_ecoevo

**[Website](https://jlw-ecoevo.github.io)** 

**[Google Scholar](https://scholar.google.com/citations?user=IaAUSiQAAAAJ&hl=en)**

**Bio:** The battle between microbes and their viruses is ancient and ongoing. Over billions of years of coevolution, microbes have evolved a great diversity of strategies to protect themselves against viral infections. What drives selection favoring a particular antiviral defense strategy? What new defense systems are still waiting to be discovered? How do defense systems coevolve with pathogens over time, and how does this coevolution feed back into the dynamics of host-virus interactions? The central theme of my previous and ongoing work is the application of tools from the fields of machine learning and complex systems science, alongside more traditional approaches from population genetics and theoretical ecology, to understand the ecology and evolution of antiviral defense strategies.

## Elizabeth McDaniel
<img src="https://avatars1.githubusercontent.com/u/18299362?s=460&u=2759d5e6ec166f8f71fb0b1521324fe813d049a2&v=4" align="left" width="200">

**Email:** elizabethmcd93@gmail.com

**Twitter:** @lizilla93

**[Website](elizabethmcd93.github.io)**

**Bio:** I am a PhD student in the Microbiology Doctoral Training Program in Dr. Katherine McMahon's lab at the University of Wisconsin - Madison. I use a combination of enrichment culture techniques and integrated genome-resolved metagenomics to investigate bacterial population dynamics of engineered wastewater systems. I am passionate about helping people solve their analytical challenges as it relates to their research. I am a certified Carpentries instructor and a co-chair of the Computational Biology, Ecology, and Evolution [(ComBEE)](https://combee-uw-madison.github.io/studyGroup/) study group on the UW-Madison campus.

## Morgan V. Evans
<img src="https://avatars2.githubusercontent.com/u/51716739?s=460&u=952436b987be5903627bc69d88a77fdd36ec1efa&v=4" align="left" width="200">

**Email:** evans.2408@osu.edu

**Twitter:** @morgvevans

**[Website](https://github.com/morgvevans)**

**[Google Scholar](https://scholar.google.com/citations?user=vVUCZw0AAAAJ&hl=en&authuser=1)**

**Bio:** I am postdoctoral researcher at The Ohio State University in the Colleges of Public Health & Veterinary Preventative Medicine. My current research uses -omic tools to investigate strain level overlap between humans, animals, and the environment, as well as to reconstruct chemical/xenobiotic microbial pathways in the human and animal gut as it relates to health & disease. My PhD work focused on microbial xenobiotic metabolisms in hydraulically fractured oil and gas systems. I am eager to help people who don't have coding or bioinformatics backgrounds learn their way around the command line for their work, particularly for metagenomics and 16S rRNA sequencing analysis.

## Joy Buongiorno

<img align="left" src="/images/joyb.jpeg">

**Email:** jbuongior21@gmail.com

**Twitter:** @DrJoyBuongiorno

**[Google Scholar](https://scholar.google.com/citations?user=3pa76GEAAAAJ&hl=en&oi=ao)**

**Bio:**  I am a geomicrobiologist combining a background in geoscience with a diverse toolkit of novel molecular methods, sequencing analysis, and bioinformatic tools to delve into the interactions between the biosphere and geosphere. I am currently a postdoctoral fellow at the Earth and Planets Laboratory at the Carnegie Institution of Washington working on the NASA Astrobiology Institute ENIGMA Project. I apply methods at the frontier of integrative earth, life, and data science to answer big questions related to the co-evolution of the biosphere and geosphere through deep time.

## Mike Lee

<img align="left" src="/images/Mike-Lee-Atlantis.jpg" width="200px" style="padding-right: 15px">

**Email:** MikeLee@bmsis.org  

**Twitter:** [@AstrobioMike](https://twitter.com/AstrobioMike){:target="_blank"}  

[Website](https://astrobiomike.github.io/research/){:target="_blank"}  

[Google Scholar](https://scholar.google.com/citations?user=-ONw6lsAAAAJ){:target="_blank"}  
<br>
**Bio:** Iâ€™m a microbial ecologist/bioinformatician at NASAâ€™s Ames Research Center in northern California, USA. I get to work in all kinds of different systems ranging from the bottoms of our oceans up to the International Space Station ðŸ‘½. Having [gone through it myself](https://astrobiomike.github.io/about/){:target="_blank"}, I very much enjoy trying to help folks get through the initial steep learning curve that lots of us face when we start getting into bioinformatics. In addition to regularly helping to organize and teach at bioinformatics workshops such as [STAMPS](https://mblstamps.github.io/){:target="_blank"} and [ANGUS](https://angus.readthedocs.io/en/2019/){:target="_blank"}, I develop and maintain [Happy Belly Bioinformatics](https://astrobiomike.github.io/){:target="_blank"}, a site that aims to be a useful resource for biologists who are new to bioinformatics ðŸ™‚  

## Ashley B. Cohen, M.S.

<img align="left" src="/images/CohenPhoto.jpg" width="150px" style="padding-right: 15px">

**Email:** ashleybrookcohen@gmail.com

**Twitter:** @ashbc1988

**Bio:**  I am a PhD candidate in biogeochemistry at the School of Marine and Atmospheric Sciences, Stony Brook University, NY in Dr. Gordon Taylorâ€™s Marine Microbial Ecology laboratory. I specialize in the geobiology, or the interaction of geochemical processes and microbes, of oxygen-free environments. My dissertation work at Fayetteville Green Lake, NY, is focused on how the succession of microbial carbon fixation pathways and sulfur metabolisms along redox gradients influences stable isotope signatures. I am especially interested in how they are compartmentalized into suspended and sinking particulate matter in the water column, and ultimately, what is incorporated into sediment. I hope this will provide insight into which early earth anoxic water column processes are reflected in the geologic record. I combine microbiology techniques including Illumina sequencing, stable isotope probing, radiotracer rate measurements, qPCR, and fluorescence in situ hybridization with geochemical techniques to answer these questions.


## Harriet Alexander

<img align="left" src="https://github.com/AlexanderLabWHOI/academic-kickstart/blob/master/static/img/headshot-01.jpg" width="200px" style="padding-right: 15px">

**Email:** halexander@whoi.edu

**Twitter:** @nekton4plankton

**[Website](https://alexanderlabwhoi.github.io/)*

**[Google Scholar](https://scholar.google.com/citations?user=FdYCw1gAAAAJ&hl=en)**

**Bio:** I am an Assistant Scientist in the Biology Department at the Woods Hole Oceanographic Institution interested in bioinformatics and microbes in the ocean. My work focuses on characterizing the the interplay of physiological ecology and diversity in the ocean with a focus on eukaryotic plankton. Much of my research has leveraged meta-omic data to derive insights into the diversity and function of these ecosystems.
