---
title: "How is population decline influenced by evolutionary mechanisms?"
excerpt: " The Flanagan Lab utilizes population genetics tools to improve our understanding of the genetic dynamics and factors that contribute to declining populations. The field of conservation genetics looks at the application of genetic/genomic data to inform and improve conservation efforts and the restoration of populations. Our lab is using a variety of molecular tools to investigate the evolutionary mechanisms impacting New Zealand populations of kororā, pipefish and vegetation impacted by dieback disease "
permalink: /research/cons-gen
collection: research
---

We are interested in studying what causes populations to decline. Variation between populations is driven by a number of evolutionary mechanisms, for example, the availability of prey species, introduction of disease, inbreeding depression and intraspecific phenotypic variation. Evaluating the ecological and evolutionary interactions within populations is important for informing effective conservation management of endemic species.

To conserve endemic species in Aotearoa, we can utilise the field of population genetics, investigating the structure of local populations and the factors that impact the genome across generations. When informing conservation management efforts, understanding and acknowledging different evolutionary mechanisms is important as they may be impacting population growth e.g. inbreeding depression. We can utilise population genetic/genomic tools to estimate effective population sizes and establish a baseline dataset on population structure, against which we can measure change over time. The Flanagan lab is exploring molecular methods for assessing Pohatu’s kororā population, with a focus on reducing the need for invasive sampling. We are particularly interested in breeding pairs where parents are a mix of the white-flippered and little blue morphotypes. We hope to utilise molecular methods to better understand the persistence of the white-flippered morphotype, which is unique to the Canterbury region and explore the impacts phylogenetic constraints may be having on the colony. Additionally, we are working on modelling the distribution of the wide-bodied pipefish, Stigmatopora nigra, and using genetic analysis to compare population structure across its range. By comparing phenotypic and genotypic data of S. nigra across Aotearoa we will evaluate different selection pressures, assess the evolutionary mechanisms that drive local populations and expand on the currently limited understanding of gene flow across the S. nigra range.

To effectively conserve valuable species we need to both identify the environmental factors contributing to regional declines and establish the underpinning evolutionary mechanisms. To this end, the Flanagan lab is currently focused on diet and disease as agents of decline. In collaboration with Mana Whenua, Pohatu Penguins and the LaRue and ConSERT labs we are utilizing long-read nanopore sequencing to assess the diet of the Pohatu kororā colony as a potential driver of local population decline. Using non-invasive faecal and moult feather samples to develop baselines for local diet, this project aims to facilitate community engagement in the genetic sampling of local populations. Collaboration with the Banks Peninsula community will be further supported through the development of remote sequencing using the portable MinION device (Oxford nanopore technologies). We are working towards a molecular dataset of the kororā diet across multiple breeding seasons. This dataset will feed into Pohatu Penguin’s research on kororā distribution and foraging range and help to highlight local risk factors that may negatively impact recruitment. Moreover, identifying important prey species is important when appraising the potential impacts of a changing climate on kororā populations. In addition to dietary analysis, the Flanagan lab is interested in the coevolution of invasive diseases, specifically Phytophthora, and the vegetation they impact. In collaboration with Claudia Meisrimler, we are looking at Phytophthora cinnamomi, which causes dieback non-discriminantly across endemic and economically important flora and is a close relative to P. agathidicida; the source of Kauri dieback. In order to tackle invasive diseases, it is important we understand the genetic mechanisms that drive them. This study will involve the construction and analysis of a P. cinnamoni genome using long-read nanopore sequencing. Improving current data on the functional genomics of  P. cinnamoni and its co-evolution with valuable vegetation will allow for better-informed management and conservation programmes aimed at tackling dieback diseases.

Photos!
-Cute penguin chick pic or photo of showing sample (Wendy and her faeces)
-Sophie Phytophthora photo coming soon (though not Phytophthora cinnamomi)


Some relevant papers include:

Howell LA, LaRue M, Flanagan SP (2021) Environmental DNA as a tool for monitoring Antarctic vertebrates. New Zealand Journal of Zoology. DOI: 10.1080/03014223.2021.1900299.
Does Claudia have relevant papers? (Alex?)
