---
title: "Monitoring antimicrobial resitance in 100K host and environmental samples"
collection: poster
type: "Poster"
permalink: /talks/2020-11-04-poster
venue: "Antimicrobial Resistance - Genomes, Big Data and Emerging Technologies"
date: 2020-11-04
location: "Online"
---
The growing amount of NGS data available in public data repositories presents a unique opportunity to explore new ways of monitoring antimicrobial resistance (AMR) across different sample types. The diversity of sample origins, locations and collection dates can provide a detailed picture of the emergence and spread of different AMR genes worldwide. We have downloaded 122 Tbp sequence reads from 116,758 host and environmental metagenomes from the European Nucleotide Archive (ENA), have quality checked and trimmed the raw sequencing reads and mapped them against two reference sequence databases: the Silva database to describe the bacterial content and ResFinder to detect AMR genes. Using compositional data analysis on the output, we have compared the results across different experimental procedures.  
We will showcase the feasibility of large-scale metagenomic AMR analysis by focusing on the mcr gene family, which confers resistance to colistin. The spread of mcr genes in recent years poses a significant threat to public health, as colistin is a last-resort against multidrug-resistant infections. By using our data, we have characterized the advancement of mcr genes across animal and human populations. In human samples, mcr-9 were more present as compared to other hosts such as livestock displayed a more diverse distribution of all the genes (p < 0.05). We also see that the mcr genes did not have a global distribution but instead were specific to each demographic region with mcr-9 being more prevalent in North American samples and mcr-1 in Asian samples (p < 0.05). Storing and analyzing large quantities of metagenomic data might not be possible for all research groups, which is why we are planning to setup a database and share it together with the results in a webservice. This will allow the scientific community to explore and use the data in new ways. Furthermore, as new metagenomic data becomes available online, we will add these samples to the database and analyze them.


![poster](../images/uk_amr_poster.png)